sqlglot/parser.py

```diff
@@ -2050,7 +2050,10 @@ def _parse_column_constraint(self):
         elif self._match(TokenType.SCHEMA_COMMENT):
             kind = self.expression(exp.CommentColumnConstraint, this=self._parse_string())
         elif self._match(TokenType.PRIMARY_KEY):
-            kind = exp.PrimaryKeyColumnConstraint()
+            is_ascending = None
+            if self._match(TokenType.ASC) or self._match(TokenType.DESC):
+                is_ascending = self._prev.text == "ASC"
```

Comment:
self._prev.token_type ==TokenType.ASC

Reply:
I keep forgetting to compare token types instead of text lol